sling.go

```diff
@@ -360,9 +360,9 @@ func (s *Sling) ReceiveSuccess(successV interface{}) (*http.Response, error) {
 // Receive creates a new HTTP request and returns the response. Success
 // responses (2XX) are JSON decoded into the value pointed to by successV and
 // other responses are JSON decoded into the value pointed to by failureV.
-// If the status code of response is 204(no content), decoding is skipped.
-// Any error creating the request, sending it, or decoding the response is
-// returned.
+// If the status code of response is 204(no content) or the Content-Lenght is 0,
```

Comment:
In case it's helpful to you, please note there's a typo in "Content-Lenght". It should be "Content-Length".